pipeline/inputs/http.md

```diff
@@ -16,6 +16,10 @@ description: The HTTP input plugin allows you to send custom records to an HTTP
 | successful_response_code | It allows to set successful response code. `200`, `201` and `204` are supported.                                                              | 201     |
 | success_header           | Add an HTTP header key/value pair on success. Multiple headers can be set. Example: `X-Custom custom-answer`                                  |         |
 
+### TLS / SSL
+
+HTTP input plugin supports TTL/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](tcp-and-tls.md) section.
```

Comment:
I don't think that is the right link as it is in the outputs section.

We probably want to link to the general section for it: https://github.com/fluent/fluent-bit-docs/blob/master/administration/transport-security.md